7.0-cli/bin/magedbm2

```diff
@@ -0,0 +1,5 @@
+#!/usr/bin/env bash
+
+[ "$DEBUG" = "true" ] && set -x
+
+su www-data -s /bin/bash -c "magedbm2.phar --root-dir=$MAGENTO_ROOT $*"
```

Comment:
1. We should `exec` here to pass the execution into another binary instead of spawning a child process and waiting for it to resolve
2. I'm pretty sure we want to use `"$@"` here instead of `$*`. The latter expands parameters into a single quoted word, while the former expands them into separate quoted words (note  the quotes around the `$@` too). More info: https://www.gnu.org/software/bash/manual/html_node/Special-Parameters.html

In the M1 containers, I ended up installing `sudo` to make these wrapper scripts easier to handle, which made the passthrough command simply:

```
exec sudo -u www-data -- n98-magerun.phar "$@"
```
([Reference](https://github.com/meanbee/docker-magento/blob/master/src/bin/magerun))